CMakeLists.txt

```diff
@@ -5,9 +5,12 @@ set(CMAKE_C_STANDARD 11)
 project(plutovg C)
 
 add_library(plutovg STATIC)
+target_compile_options(plutovg PUBLIC -fsanitize=fuzzer-no-link)
```

Comment:
Since you already have built infrastructure under the fuzz directory, I don't see any need to modify the top level CMakeLists.

Reply:
I needed to add instrumentation to the library to support fuzzing, which would make sense to do in the root CMakeLists file. I have, in my later submissions, learned to make this optional with a CMake build option and have updated my code to reflect this. Thus, a default build will be unaffected by my changes but specifying -DBUILD_FUZZER=1 will instrument the binary and build the fuzzer directory.